tests/ijby/test-sort.py

```diff
@@ -1016,6 +1016,36 @@ def test_sort_double_negation():
     assert_equals(DT[:, :, sort(f.A)], RES4)
 
 
+#-------------------------------------------------------------------------------
+# Sort with positional value for NAs
+#-------------------------------------------------------------------------------
+
+def key_func(x, rev, na_pos):
+    if na_pos == "first":
+        return not rev if x is not None else rev
+    else:
+        return not rev if x is None else rev
```

Comment:
Simplifies into
```suggestion
    return (x is None) ^ (rev) ^ (na_pos == "first")
```

It's not a big deal of course, but such a rare opportunity to use operator XOR in real life, I thought it would be a shame to pass it up :)

Reply:
I will grant your wish :)